cloudformation.yml

```diff
@@ -85,7 +85,9 @@ Resources:
             Topic: !Ref ChoiceMadeSnsTopic
       Environment:
         Variables:
-          CHOICE_TABLE_NAME: !Ref ChoiceTable
+          DYNAMODB_TABLE_NAME: !Ref ChoiceTable
+          DYNAMODB_ENDPOINT:
```

Comment:
This is set under `Globals`, shouldn't be needed here.

Reply:
Good catch. Fixed.